charts/azure-api-management-gateway/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "0.1"
+description: A Helm chart to deploy an Azure API Management Gateway on Kubernetes
+name: azure-api-management-gateway
```

Comment:
This is just a suggestion but Helm chart name can be changed to something like `azure-api-management-self-hosted-gateway` but might be a bit long